README.md

```diff
@@ -1 +1,53 @@
-# github-mirror
+# GitHub Mirror
+
+Implements [conditional requests](https://developer.github.com/v3/#conditional-requests)
+
+Check the code:
+
+```
+$  make check
+```
+
+Run the development server:
+
+```
+$  make run-app
+```
+
+Using:
+
+```
+$ pipenv run python
+Python 3.6.10 (default, Dec 20 2019, 00:00:00)
+[GCC 9.2.1 20190827 (Red Hat 9.2.1-1)] on linux
+Type "help", "copyright", "credits" or "license" for more information.
+>>>
+>>> import requests
+>>> requests.get('http://localhost:8080/repos/app-sre/github-mirror')
+<Response [200]>
+>>>
+>>> requests.get('http://localhost:8080/repos/app-sre/github-mirror')
+<Response [200]>
+>>>
+>>> requests.get('http://localhost:8080/stats').json()
+{'cache_hit': 1, 'cache_miss': 1}
+```
+
+Notice that the `/stats` endpoint is not considered for cache statistics.
```

Comment:
what do you mean exactly with this?

Reply:
`GET` to `/stats` won't increase the `cache_miss`/`cache_hit` counters.

Reply:
ahh, ok 👍